assets/js/atomic/components/product-list/summary/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const ProductListSummary = ( { className, product } ) => {
+	if ( ! product.description ) {
+		return null;
+	}
+
+	return (
+		<div
+			className={ classnames(
+				className,
+				'wc-block-grid__product-summary'
+			) }
+			dangerouslySetInnerHTML={ {
+				__html: product.description,
+			} }
+		/>
+	);
+};
+
+ProductListSummary.propTypes = {
+	className: PropTypes.string,
+	product: PropTypes.object.isRequired,
```

Comment:
Something to think about for potential future work is to have actual Product entities client side.   That way we could type check for them ( eg`PropTypes.instanceOf( WooProduct )` ) and also have an interface for getting product data (i.e. sanitized description )